kvdb-rocksdb/src/lib.rs

```diff
@@ -443,15 +443,22 @@ impl Database {
 							stats_total_bytes += key.len();
 							batch.delete_cf(cf, &key).map_err(other_io_err)?
 						}
-						DBOp::DeletePrefix { col: _, prefix } => {
+						DBOp::DeletePrefix { col, prefix } => {
 							if prefix.len() > 0 {
-								let end_range = kvdb::end_prefix(&prefix[..]);
-								batch.delete_range_cf(cf, &prefix[..], &end_range[..]).map_err(other_io_err)?;
+								if let Some(end_range) = kvdb::end_prefix(&prefix[..]) {
+									batch.delete_range_cf(cf, &prefix[..], &end_range[..]).map_err(other_io_err)?;
+								} else {
+									for (key, _) in self.iter_from_prefix(col, &prefix[..]) {
+										batch.delete_cf(cf, &key[..]).map_err(other_io_err)?;
+									}
```

Comment:
I see that `delete_range` accepts only ["begin_key", "end_key"), but maybe there is a more efficient way?

Reply:
Maybe, will do some research next week.